server/src/config/trulioo.config.js

```diff
@@ -1,16 +1,16 @@
 import axios from 'axios';
 
-const TRULIOO_BASE_URL = process.env.TRULIOO_PRODUCTION === "true" ?
-  'https://gateway.trulioo.com/live' : 'https://gateway.trulioo.com/trial';
+const TRULIOO_BASE_URL = process.env.TRULIOO_PRODUCTION === 'true' ? 'https://gateway.trulioo.com/live' : 'https://gateway.trulioo.com/trial';
 export const ASYNC_CALLBACK_URL = `api/send-check-result`;
 export const EXTERNAL_SERVER_URL = `${process.env.EXTERNAL_SERVER_URL}`;
 export const EMBED_ID_TOKEN_URL = `https://api-gateway-admin.trulioo.com/embedids/tokens`;
 export const EXPERIENCE_TRANSACTION_URL = 'https://gateway.trulioo.com/experienceTransaction';
 export const truliooApiKey = process.env.X_TRULIOO_API_KEY;
 export const truliooPublicKey = process.env.TRULIOO_PUBLIC_KEY;
+export const ipListWhitelist = process.env.IP_LIST_WHITELIST.split(',');
```

Comment:
Can we please call this ipAddressAllowList instead? It's more descriptive and also I think it's important not to continue the white=good pattern and be more culturally-sensitive instead.

Otherwise this PR looks good, as long as it's true that we'll know which addresses to put in that array.